lib/game.rb

```diff
@@ -81,21 +77,32 @@ def player_ship_placement(ship, show)
         example = "(ie. A1, A2)"
     end
       show == 0 ? (puts @player_board.render) : (puts @player_board.render(true))
-
-          puts " \n Enter the squares for the #{ship.name} (#{ship.length} spaces #{example}): \n "
-      
-
-       
-          user_input = gets.chomp.upcase.gsub(",", " ").split
+        puts " \n Enter the coordinates for the #{ship.name} (#{ship.length} spaces #{example}): \n "
+        user_input = gets.chomp.upcase.gsub(",", " ").split
 
       until @player_board.valid_placement?(ship, user_input)
-          puts "Invalid placement. Must be consecutive and not diagonal"
-          user_input = gets.chomp.upcase.gsub(",", " ").split
+        coord_statement(user_input, ship)
+        #   puts "Invalid placement. Must be consecutive and not diagonal"
+        user_input = gets.chomp.upcase.gsub(",", " ").split
       end
       @player_board.place_ship(ship, user_input)
       puts " \n Your #{ship.name} has been placed. \n "
   end
 
+  def coord_statement(input, ship)
```

Comment:
ran into a big last night with ship placement, when you pick cells that exist off the board it will end the runner file, it looks like this is addressing that bug, I will test after I pull down